test/functional/apps/discover/_field_visualize.ts

```diff
@@ -1,159 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * and the Server Side Public License, v 1; you may not use this file except in
- * compliance with, at your election, the Elastic License or the Server Side
- * Public License, v 1.
- */
-
-import expect from '@kbn/expect';
-import { FtrProviderContext } from '../../ftr_provider_context';
-
-export default function ({ getService, getPageObjects }: FtrProviderContext) {
```

Comment:
The test had checks for preserving query & filters while moving from Discover to Visualize: do we need to check the same behavior for Lens now?

Reply:
Had a quick look here, and think this should be adapted to Lens.

Reply:
This test already exists, it is on x-pack! `discover/visualize_field` test

Reply:
Excellent! Thanks for the highlight!
Resolving this 👍